Libraries/Components/Touchable/TouchableOpacity.js

```diff
@@ -105,12 +107,12 @@ var TouchableOpacity = React.createClass({
   },
 
   touchableHandleActivePressOut: function() {
-    this.setOpacityTo(1.0);
+    var childStyle = flattenStyle(this.refs[CHILD_REF].props.style || {});
```

Comment:
can you put the || {} outside of the flattenStyle call? flattenStyle breaks early if it's undefined